core/src/main/java/org/apache/accumulo/core/util/LocalityGroupUtil.java

```diff
@@ -50,16 +52,16 @@
 
   // private static final Logger log = Logger.getLogger(ColumnFamilySet.class);
 
-  public static final Set<ByteSequence> EMPTY_CF_SET = Collections.emptySet();
+  public static final Set<ByteSequence> EMPTY_CF_SET = ImmutableSet.of();
```

Comment:
Oh, like the reference check against `this`? I could see that being beneficial.

A comment in the code as to use for ImmutableSet for that reason would be quite appreciated.